Automattic/wp-calypso
# WIP: ThemeCollections: Add/theme collection add see all

Related to https://github.com/Automattic/dotcom-forge/issues/3959

## Proposed Changes

* Add a `See all` link in the Theme Collection

## Testing Instructions



- Use the Calypso live link and go to /themes in incognito
- Enable the feature flag: flags=themes/discovery-lots
- Notice a `See all` link on the page
- Click on the Premium one and on the Partner one
- Notice that you are redirected to /themes/premium and /themes/marketplace
## Pre-merge Checklist
